model_api/python/docs/ActionClassification.md

```diff
@@ -0,0 +1,79 @@
+# ActionClassification Wrapper
+
+## Use Case and High-Level Description
+
+The `ActionClassificationModel` is a wrapper class designed for action classification models.
+This class provides support for data preprocessing and postprocessing like other model wrapper classes.
+Note that it isn't a subclass of `ImageModel`. It gets video as input so it is different than ImageModel.
+
+## How to use
+
+Utilizing the ActionClassificationModel is similar to other model wrappers, with the primary difference being the preparation of video clip inputs instead of single images.
+
+Below is an example demonstrating how to initialize the model with OpenVINO IR files and classify actions in a video clip.
+
+
+```python
+import cv2
+import numpy as np
+# import model wrapper class
+from model_api.models import ActionClassificationModel
+# import inference adapter and helper for runtime setup
+from model_api.adapters import OpenvinoAdapter, create_core
+
+
+# load video and make a clip as input
+cap = cv2.VideoCapture("sample.mp4")
+input_data = np.stack([cap.read()[1] for _ in range(8)])
```

Comment:
Do we always have to use 8 frames per clip?

Reply:
no it depends on user's configuration. In real scenario, user needs to sample # clips from video.

Reply:
Right, but what is the #? In the real application users could run several different IRs with that wrapper, each of the IRs could have a different # hardcoded as a static shape.
I propose to add 'getClipSize() -> int' method to the wrapper. It should read # from the static input shape and return it. If the input shape is dynamic, the method could return -1 (as a sign of dynamic shape in OV).

Reply:
\# means num_clips user set. Actually num_clips is already stored as an attribute here
```python
class ActionClassificationModel(Model):
    def __init__(...):
        ...
        if self.nscthw_layout:
            self.n, self.s, self.c, self.t, self.h, self.w = self.inputs[
                self.image_blob_name
            ].shape
        else:
            self.n, self.s, self.t, self.h, self.w, self.c = self.inputs[
                self.image_blob_name
            ].shape
``` 
`self.t` is num_clips.

Reply:
Great, then that'd be easy to implement the method & mention it in the documentation

Reply:
done https://github.com/openvinotoolkit/model_api/pull/178/commits/4708d2bcbd61e6d1e911161132af885cbdab210f